test/new-e2e/tests/installer/all_packages_test.go

```diff
@@ -0,0 +1,209 @@
+		datadogConfig["dd_url"] = s.Env().FakeIntake.URL
+		datadogConfig["skip_ssl_validation"] = true
+		datadogConfig["apm_config"] = map[string]interface{}{
+			"apm_dd_url": s.Env().FakeIntake.URL,
+		}
+	}
+	rawDatadogConfig, err := yaml.Marshal(datadogConfig)
+	require.NoError(s.T(), err)
+	s.Env().RemoteHost.MustExecute("sudo mkdir -p /etc/datadog-agent")
+	s.Env().RemoteHost.MustExecute(fmt.Sprintf("echo '%s' | sudo tee /etc/datadog-agent/datadog.yaml", string(rawDatadogConfig)))
+	s.Env().RemoteHost.MustExecute("sudo chown -R dd-agent:dd-agent /etc/datadog-agent")
+}
+
+func (s *packageBaseSuite) InstallAgentPackage() {
+	env := map[string]string{
+		"DD_API_KEY":                 apiKey(),
+		"DD_SITE":                    "datadoghq.com",
+		"DD_INSTALLER_REGISTRY":      "669783387624.dkr.ecr.us-east-1.amazonaws.com",
+		"DD_INSTALLER_REGISTRY_AUTH": "ecr",
+	}
+	s.Env().RemoteHost.MustExecute(`sudo -E datadog-installer install oci://669783387624.dkr.ecr.us-east-1.amazonaws.com/agent-package:`+fmt.Sprintf("pipeline-%v", os.Getenv("CI_PIPELINE_ID")), client.WithEnvVariables(env))
+	s.Env().RemoteHost.MustExecute(`timeout=30; unit=datadog-agent.service; while ! systemctl is-active --quiet $unit && [ $timeout -gt 0 ]; do sleep 1; ((timeout--)); done; [ $timeout -ne 0 ]`)
+}
+
+// TODO: This is a hack to install a working version of apm-inject
+func (s *packageBaseSuite) InstallInjectorPackageTemp() {
+	env := map[string]string{
+		"DD_API_KEY":                 apiKey(),
+		"DD_SITE":                    "datadoghq.com",
+		"DD_INSTALLER_REGISTRY":      "669783387624.dkr.ecr.us-east-1.amazonaws.com",
+		"DD_INSTALLER_REGISTRY_AUTH": "ecr",
+	}
+	s.Env().RemoteHost.MustExecute(`echo "DD_APM_RECEIVER_SOCKET=/var/tmp/apm.socket" | sudo tee -a /etc/environment`)
+	if s.Env().FakeIntake != nil {
+		s.Env().RemoteHost.MustExecute(fmt.Sprintf(`echo "DD_APM_DD_URL=%s" | sudo tee -a /etc/environment`, s.Env().FakeIntake.URL))
+	}
+	s.Env().RemoteHost.MustExecute("sudo mkdir -p /etc/systemd/system/datadog-agent-trace.service.d")
+	s.Env().RemoteHost.MustExecute(`printf "[Service]\nEnvironmentFile=-/etc/environment\n" | sudo tee /etc/systemd/system/datadog-agent-trace.service.d/inject.conf`)
+	s.Env().RemoteHost.MustExecute("sudo systemctl daemon-reload")
+	s.Env().RemoteHost.MustExecute("sudo -E datadog-installer install oci://669783387624.dkr.ecr.us-east-1.amazonaws.com/apm-inject-package:pipeline-34163111", client.WithEnvVariables(env))
+}
+
+func (s *packageBaseSuite) InstallPackageLatest(pkg string) {
+	env := map[string]string{
+		"DD_API_KEY": apiKey(),
+		"DD_SITE":    "datadoghq.com",
+	}
+	s.Env().RemoteHost.MustExecute(fmt.Sprintf(`sudo -E datadog-installer install oci://gcr.io/datadoghq/%s-package:latest`, strings.TrimPrefix(pkg, "datadog-")), client.WithEnvVariables(env))
+}
+
+func (s *packageBaseSuite) Purge() {
+	s.Env().RemoteHost.MustExecute("sudo apt-get remove -y --purge datadog-installer || sudo yum remove -y datadog-installer")
```

Comment:
You might need to add the equivalent command for `Suse` that would be using `zypper` when you want to run e2e tests on Suse